config/500-controller.yaml

```diff
@@ -0,0 +1,75 @@
+# Copyright 2019 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ceph-source-controller
+  namespace: knative-ceph
```

Comment:
If we keep this source in the `knative-source` namespace (similar to the other sources), we could also simplify the controller name to just `ceph-controller`

Reply:
I modified to `knative-source` namespace. and renamed the different entities to not include the term "source" in them.
wasn't sure about cluster roles as they are not namespaced - but I made the same change there.